auth/debug.go

```diff
@@ -26,7 +26,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// Logging HTTP transport
+// Package auth provides HTTP transport logging
```

Comment:
yes, i started on this and moved the comment to auth/google.go since it seemed more appropriate than in auth/debug.go, but in any case, some accurate package comment would be nice.